### DISCUSSION TITLE
Terms to add to the glossary

### DISCUSSION THREAD

**Discussion by han-so1omon** [icon-community/icon.community#10] [General]
## intro
Add new terms as you think of them

## some terms

Oracle - An oracle is a bridge between the [blockchain](https://ethereum.org/en/glossary/#blockchain) and the real world. They act as on-chain [APIs](https://ethereum.org/en/glossary/#api) that can be queried for information and used in [smart contracts](https://ethereum.org/en/glossary/#smart-contract)

Relay - A component of a cross-chain messaging system responsible for networking between blockchains and assisting in the aggregation of proofs to guarantee verifiable results.

**Comment by han-so1omon** [comment]
Instruction fee - A charge that must be paid in association with executing a computational instruction using computational power on the blockchain

**Comment by han-so1omon** [comment]
Interoperable - Able to utilize multiple, independently run and validated blockchains to create a single application

Cross-chain messages - Calls from one blockchain to another

**Comment by han-so1omon** [comment]
Trustless - A promise that can be made between multiple parties without the requirement for any one party to simply believe that the other parties are telling the truth

**Comment by han-so1omon** [comment]
Personally Identifiable Information (PII) - Any information that permits the identity of an individual to be directly or indirectly inferred, including any information that is linked or linkable to that individual

**Comment by han-so1omon** [comment]
Open source - Something people can modify and share because its design is publicly accessible

Free and open source - Something where anyone is freely licensed to use, copy, study, and change it in any way, and the products are openly shared so that people are encouraged to voluntarily improve the design and functionality

**Comment by han-so1omon** [comment]
Step - A unit of measurement associated with computational processing on the ICON network. Currently it's economic value is associated to 1 * 10^-8 ICX

**Comment by han-so1omon** [comment]
On-chain - Data or logic that is codified into the blockchain global state machine

Off-chain - Data or logic that is not on the blockchain

Global state machine - A logic system that tracks current status and facilitates changes in status through decisions and transition processes

**Comment by DavidFBD** [comment]
Suggestions have been added to icon.community/glossary
